app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -136,11 +138,17 @@ firewall --disable
 <% else -%>
 firewall --<%= os_major >= 6 ? 'service=' : '' %>ssh
 <% end -%>
+
+<% if host_param_true?('disable-kdump') -%>
```

Comment:
I'm thinking if we should expose the options directly. https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/10-beta/html/automatically_installing_rhel/kickstart-commands-and-options-reference#addon-comredhatkdump mentions there are several. Perhaps just expose some `kdump-options` parameter that you can set to `--disable`?

Reply:
Ok, should it be passed as a string or should I try to validate the input?

Reply:
@ekohl switched to kdump-options, where should I check for template validation?

Reply:
I'd keep it simple and pass it as a string.